SMLHelper/Utility/EnumCacheManager.cs

```diff
@@ -131,9 +131,9 @@ internal void SaveCache()
             }
             catch (Exception exception)
             {
-                Logger.Log("Caught exception when saving cache!");
-                Logger.Log("Exception message: " + exception.Message);
-                Logger.Log("StackTrace: " + Environment.NewLine + exception.StackTrace);
+                Logger.Log("Caught exception when saving cache!", LogLevel.Error);
```

Comment:
Can be replaced with a single log entry.
```CSharp
Logger.Log($"Caught exception when saving cache!{Environment.NewLine}{exception.}", LogLevel.Error);
// ToString() is called implicitly when in an interpolated string
```
https://docs.microsoft.com/en-us/dotnet/api/system.exception.tostring?view=netframework-4.7.2
> ToString returns a representation of the current exception that is intended to be understood by humans. Where the exception contains culture-sensitive data, the string representation returned by ToString is required to take into account the current system culture. Although there are no exact requirements for the format of the returned string, it should attempt to reflect the value of the object as perceived by the user.